Makefile.common

```diff
@@ -28,7 +28,7 @@ path_to_add := $(addsuffix /bin,$(subst :,/bin:,$(GOPATH))):$(PWD)/tools/bin
 export PATH := $(path_to_add):$(PATH)
 
 GO              := GO111MODULE=on go
-BUILD_FLAG	:= -tags codes
+BUILD_FLAG	:= -pgo=default.pgo -tags codes
```

Comment:
Where is ```default.pgo``` ? we need to add the document about it.

Reply:
Which document is recommended to add this?

Reply:
How about adding a comment about a desc of default.pgo?